blocitoff/Gemfile

```diff
@@ -1,4 +1,7 @@
 source 'https://rubygems.org'
+
+#in the terminal I 'ran gem install faker'
```

Comment:
Cool! Just so you know, when you add a gem to your `Gemfile` you don't need to run `gem install` from the terminal as well. Running `bundle install` will fetch those gems for you.